### ISSUE TITLE
Refactor tests to not use --debug

### ISSUE THREAD

**Issue by sebastianbergmann** [sebastianbergmann/phpunit#4704]
The `--debug` CLI option is deprecated. The following tests use it for testing something that be tested in a better way:

* tests/end-to-end/coverage-annotation-based-filter/covers-annotation-based-filter.phpt
* tests/end-to-end/coverage-annotation-based-filter/uses-annotation-based-filter.phpt
* tests/end-to-end/execution-order/defects-first-order-via-cli.phpt
* tests/end-to-end/execution-order/execution-order-options-via-config.phpt
* tests/end-to-end/execution-order/order-by-duration-via-cli.phpt
* tests/end-to-end/execution-order/order-by-duration-via-phpunit-xml.phpt
* tests/end-to-end/execution-order/test-order-randomized-seed-with-dependency-resolution.phpt
* tests/end-to-end/execution-order/test-order-reversed-with-dependency-resolution.phpt
* tests/end-to-end/execution-order/test-order-reversed-without-dependency-resolution.phpt
* tests/end-to-end/execution-order/test-order-size-with-dependency-resolution.phpt
* tests/end-to-end/loggers/custom-printer-debug.phpt
* tests/end-to-end/regression/GitHub/1216.phpt
* tests/end-to-end/regression/GitHub/1330.phpt
* tests/end-to-end/regression/GitHub/322.phpt
* tests/end-to-end/regression/GitHub/3396/issue-3396-test.phpt

**Comment by arderyp** [comment]
is there a replacement for the `--debug` option, or is more verbose output being abandoned altogether?

**Comment by sebastianbergmann** [comment]
`--trace-text` is the more useful (and even more verbose) replacement.

**Comment by arderyp** [comment]
good to know, thanks!

**Comment by arderyp** [comment]
will the `--verbose` option remain?

**Comment by sebastianbergmann** [comment]
> will the `--verbose` option remain?

For now there are no plans to remove or change `--verbose`. However, thinking about it just now, I always set `verbose="true"` in my configuration files, because I do not want to miss the additional information. Maybe this is a sign to make the output controlled by this flag non-optional in the future.

**Comment by arderyp** [comment]
@sebastianbergmann the option [you suggested](https://github.com/sebastianbergmann/phpunit/issues/4704#issuecomment-857366680) `--trace-text` does now exist in 10.0.5. It also appears `--verbose` was indeed removed.

I see options like '--display-warnings` and `--display-errors`, but I see no `--display-all`.

Is there some new way, either a command line flag or phpunit.xml config. to simply tell phpunit 10 to give me a full verbose output?

**Comment by sebastianbergmann** [comment]
I do not think that a `--display-all` option would be useful.

**Comment by arderyp** [comment]
@sebastianbergmann so there is no way for me to provide an argument or phpunit.xml config to display full verbose output? I simply have to pass in each oy the many `--display-` manually, every time I run phpunit... there's no other full verbosity option? :confused:

**Comment by arderyp** [comment]
I set up a composer command to automatically include all of the `--display-*` options, as a handy workaround.

That said, I had custom extension logic that provided my own verbose output if/when `--debug` was passed as an argument, but now passing it gives an error since it's been removed from phpunit.  I'm just kind of brainstorming an alternate approach.

for what it's worth `--trace-text` doesn't work, and I'm guessing it was removed in phpunit 10
